src/model/extract-ga.ts

```diff
@@ -26,13 +25,30 @@ const getCommissioningDesk = (
 	return (tag && tag.title) || '';
 };
 
+const convertToCAPIPillar = (theme: CAPITheme): LegacyPillar => {
```

Comment:
`convertToLegacyPillar`? (Just thinking to align with naming of type.)

Reply:
+1 this idea!

Reply:
Great catch. Missed this when I did "rename variable"